go/pserver/cclient/CMakeLists.txt

```diff
@@ -9,5 +9,14 @@ project(cxx_go C Go)
 include(golang)
 include(flags)
 
-go_library(client STATIC)
+go_library(paddle_cclient STATIC)
```

Comment:
maybe named with `pserver_cclient` will be better? since it is only used by the new trainer module and communicate with parameter server.  `paddle_cclient` would lead to an inference client at first glance.